src/FSharp.Core/async.fsi

```diff
@@ -19,14 +19,14 @@ namespace Microsoft.FSharp.Control
     ///  This type has no members. Asynchronous computations are normally specified either by using an async expression
     ///  or the static methods in the <see cref="T:Microsoft.FSharp.Control.FSharpAsync`1"/> type.
     ///
-    ///  See also <a href="https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/asynchronous-workflows">F# Language Guide - Async Workflows</a>.
+    ///  See also <a href="https://learn.microsoft.com/en-us/dotnet/fsharp/language-reference/asynchronous-workflows">F# Language Guide - Async Workflows</a>.
```

Comment:
This link (which appears in few other places in this file) does not work for me. Not quite sure what should be the correct one, but I am guessing this one: https://learn.microsoft.com/en-us/dotnet/fsharp/language-reference/async-expressions